flaskbb/forum/models.py

```diff
@@ -27,48 +27,47 @@
 
 moderators = db.Table(
     'moderators',
-    db.Column('user_id', db.Integer(), db.ForeignKey('users.id'),
+    db.Column('user_id', db.Integer(),
+              db.ForeignKey('users.id', ondelete="CASCADE"),
               nullable=False),
     db.Column('forum_id', db.Integer(),
-              db.ForeignKey('forums.id', use_alter=True,
```

Comment:
Asked in chat, but it's worth putting here too: Why stop with `use_alter`?

Reply:
SQLAlchemy 1.0.0 added automatic resolution of foreign keys in typical use cases (http://docs.sqlalchemy.org/en/latest/core/constraints.html#sqlalchemy.schema.ForeignKeyConstraint.params.use_alter) We only have one 'untypical' use case [here](https://github.com/sh4nks/flaskbb/blob/master/flaskbb/forum/models.py#L153).